.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "BridgeApp"]
 	path = BridgeApp
 	url = https://github.com/Sage-Bionetworks/BridgeApp-Apple-SDK.git
+[submodule "SageResearch"]
```

Comment:
will this end up with two copies of SageResearch in this tree (one under BridgeApp)?

Reply:
never mind, I think I figured out why you're doing that.

Reply:
I didn't remove our validation app b/c I want to be able to use it to get data to Megha, but this does not reference that version.